Makefile

```diff
@@ -1,14 +1,28 @@
-.PHONY: all doc doc-man doc-latex docclean
+.PHONY: all doc docman doclatex docclean help
 
-doc:
+# List all platforms
+pfs := $(foreach dir,$(wildcard platforms/*/Makefile),$(subst Makefile, , $(dir)))
+
+all: 		## Build all platforms
```

Comment:
Comment should mention it's the default target it would appear on make help command.